templates/org/aac/aac-article-title-content.md

```diff
@@ -1,141 +1,158 @@
-> The H1 title is the same as the title metadata. Don't enter it here, but as the **name** value in the corresponding YAML file.
-
-_Brief introduction goes here (1-3 sentences, one paragraph only)._ [**Deploy this scenario**.](#deploy-this-scenario)
+<!--Use the aac-browse-header.yml-->
+<!--
```

Comment:
@ckittel - Note that this edit removes the "Deploy this scenario" bookmark link because we couldn't figure out why it was there. If there's a good reason to keep it, we're happy to replace it.